deploy/charts/cert-manager/values.yaml

```
@@ -928,66 +928,66 @@ cainjector:
     readOnlyRootFilesystem: true
 
   podDisruptionBudget:
-    # Enable or disable the PodDisruptionBudget resource
+    # Enable or disable the PodDisruptionBudget resource.
     #
     # This prevents downtime during voluntary disruptions such as during a Node upgrade.
     # For example, the PodDisruptionBudget will block `kubectl drain`
     # if it is used on the Node where the only remaining cert-manager
     # Pod is currently running.
     enabled: false
 
-    # Configures the minimum available pods for disruptions. Can either be set to
+    # It configures the minimum available pods for disruptions. It can either be set to
```

suggestion: It doesn't feel right to start with "it" here - it doesn't feel like it matches the style we've used elsewhere.

```suggestion
    # `minAvailable` configures the minimum available pods for disruptions. It can either be set to
```